contributors/design-proposals/propagation.md

```diff
@@ -109,6 +125,12 @@ Opinion against this:
 
 1. (From @jonboulle) May cause cross-runtime compatibility issue.
 
+1. It's not possible to validate a pod + mount propagation. Mount propagation
```

Comment:
At the time of validation of a pod the PVCs it uses may not be bound to a PVs yet, so there is nothing to validate. Binding to PVs can happen later.